app/bundles/CoreBundle/Helper/Serializer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * @copyright   2019 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Helper;
+
+class Serializer
+{
+    /**
+     * Unserializing a string can be a security vulnerability as it can contain classes that can execute a PHP code.
+     * PHP >=7 has the `['allowed_classes' => false]` option to disable classes altogether or whitelist those needed.
+     * PHP <7 do not accept the second parameter, throw warning and return false so we have to handle it diffenetly.
+     * This helper method is secure by default and handle all PHP versions.
```

Comment:
I believe this is misleading. This method is not secure by default in php < 7. We should note that here.